app/routes/events/components/EventDetail/index.tsx

```diff
@@ -51,6 +51,9 @@ type InterestedButtonProps = {
   isInterested: boolean;
 };
 
+const MIN_USER_GRID_ROWS = 2;
+const MAX_USER_GRID_ROW = 2;
```

Comment:
It's just too annoying that you don't have an `S` here that I have to comment on it :grimacing:

Reply:
oops, ye that annoys me as well